tests/parametric/test_otel_sdk_interoperability.py

```diff
@@ -0,0 +1,283 @@
+import json
+
+import pytest
+
+from ddapm_test_agent.trace import Span, root_span
+
+from utils import bug, missing_feature, irrelevant, context, scenarios, features
+from utils.parametric.spec.otel_trace import SK_INTERNAL, SK_SERVER, OtelSpan
+from utils.parametric.spec.trace import find_trace_by_root, find_span
+
+# this global mark applies to all tests in this file.
+#   DD_TRACE_OTEL_ENABLED=true is required in the tracers to enable OTel
+#   CORECLR_ENABLE_PROFILING=1 is required in .NET to enable auto-instrumentation
+pytestmark = pytest.mark.parametrize(
+    "library_env", [{"DD_TRACE_OTEL_ENABLED": "true", "CORECLR_ENABLE_PROFILING": "1",}],
+)
+
+TEST_TRACE_ID = "ff0000000000051791e0000000000041"
+TEST_SPAN_ID = "ff00000000000516"
+TEST_TRACESTATE = "dd=t.dm:-0"
+TEST_ATTRIBUTES = {"arg1": "val1"}
+
+
+@features.f_otel_interoperability
+@scenarios.parametric
+class Test_Otel_SDK_Interoperability:
+    @staticmethod
+    def assert_span_link(trace):
+        assert len(trace) == 1
+        root = root_span(trace)
+        span_links = json.loads(root["meta"]["_dd.span_links"])
+        assert len(span_links) == 1
+        link = span_links[0]
+        assert link["trace_id"] == TEST_TRACE_ID
+        assert link["span_id"] == TEST_SPAN_ID
+        assert link["trace_state"] == TEST_TRACESTATE
```

Comment:
The tracestate in span links may contain other fields (ex: sampling priority or the last Datadog span id). Instead of asserting on the whole tracestate we can check to see if it contains the sampling decision field 
```suggestion
        assert "t.dm:0" in TEST_TRACESTATE
```